docs/metrics.md

```diff
@@ -130,7 +130,11 @@ Following metrics record stats around activation handling within Controller
 Aggregate metrics for inflight activations.
 
 * `openwhisk.histogram.loadbalancer<controllerId>_activationsInflight_count` (histogram) - Records the number of activations being worked upon for a given controller. As a histogram it would give a distribution of inflight activation count within a flush interval.
-* `openwhisk.histogram.loadbalancer<controllerId>_memoryInflight_count` (histogram) - Records the amount of RAM memory in use for in flight activations. This is not actual runtime memory but the memory specified per action limits.
+* `openwhisk.histogram.loadbalancer<controllerId>_memory<invokerType>Inflight_count` (histogram) - Records the amount of RAM memory in use for in flight activations. This is not actual runtime memory but the memory specified per action limits. `invokerType` defines whether it is a managed or a blackbox action.
```

Comment:
I believe `invokerType` would have a pre defined set of possible values. Can you enumerate those in the first reference.

Reply:
invokerType has only `managed` and `blackbox` as possible values. I have made reference to these where ever invokerType is a part of the metrics.